local/protocol/translate.go

```diff
@@ -4,6 +4,8 @@ import (
 	"github.com/scootdev/scoot/runner"
 )
 
+// TODO(dbentley): we should go generate with command protoc local.proto --go_out=plugins=grpc:.
+
```

Comment:
typically when I put TODOs like this in code, I'll file a jira and also link the number.  Do we want to do that as well.  Super useful to know if it was actually ever done or punted on.  Ensures we don't lose track of tasks